integration/querier_remote_read_test.go

```diff
@@ -152,7 +152,9 @@ func runTestPushSeriesForQuerierRemoteRead(t *testing.T, c *e2emimir.Client, que
 
 func TestQuerierStreamingRemoteRead(t *testing.T) {
 	var (
-		now           = time.Now().Truncate(time.Minute) // Make assertions easier to debug.
+		// This test runs with a fixed time so that assertions are stable. The reason is that the formula used by TSDB
+		// to cut chunks has some rounding which may cause this test to be flaky if run with a random "now" time.
+		now           = must(time.Parse(time.RFC3339, "2024-01-01T00:00:00Z"))
```

Comment:
Note to reviewers: I intentionally picked an "old" timestamp to ensure the test can successfully run.